gadget/install/install.go

```diff
@@ -232,6 +232,15 @@ func createPartitions(model gadget.Model, gadgetRoot, kernelRoot, bootDevice str
 
 	// auto-detect device if no device is forced
 	// device forcing is used for (spread) testing only
+	if bootDevice == "" {
+		foundDisk, err := disks.DiskFromMountPoint("/run/mnt/ubuntu-seed", nil)
+		if err == nil {
```

Comment:
(nitpick) I would invert the order here, i.e. `if err != nil {` first as it feels more idiomatic go to me (but okay to keep of course, this is really a corner case)